### PR TITLE
Admin server for adding / deleting domains

### PR DESCRIPTION
**Pull request by gdbelvin**
This PR creates a basic Key Transparency administration API.  

This is part of supporting multi-tenancy #534

### CODE REVIEW

File: core/adminserver/admin_server.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+)
+
+type server struct {
+	storage adminstorage.Storage
+	client  trillian.TrillianAdminClient
+	keygen  keys.ProtoGenerator
+}
+
+// New returns a KeyTransparencyAdminService implementation.
+func New(
```
Comment (cesarghali) [inline core/adminserver/admin_server.go]: Please combine the next few lines into one, or only two lines. I don't think this is very readable. Thanks.

File: core/crypto/vrf/p256/p256.go [hunk 2]
```diff
@@ -37,9 +37,10 @@ import (
 	"fmt"
 	"math/big"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/google/keytransparency/core/crypto/vrf"
 	"github.com/google/trillian/crypto/keys"
+
```
Comment (cesarghali) [inline core/crypto/vrf/p256/p256.go]: I think we should come up with a strategy to group imports :)

File: core/adminserver/admin_server.go (hunk 1 above)
Comment (gdbelvin) [reply core/adminserver/admin_server.go]: Done.  Also renamed `trillian` to `tpb` to make the line shorter.

File: core/crypto/vrf/p256/p256.go (hunk 2 above)
Comment (gdbelvin) [reply core/crypto/vrf/p256/p256.go]: This follows the following import strategy:
1. golang imports
2. KT and Trillian imports
3. Third party imports
4. Underscore and renamed imports

**Pull request merged**